include/graphit/midend/mir.h

```diff
@@ -909,10 +926,11 @@ namespace graphit {
         };
 
         struct EdgeSetApplyExpr : public ApplyExpr {
-            std::string from_func = "";
-            std::vector<std::string> fromFuncFunctorArgs;
-            std::string to_func = "";
-            std::vector<std::string> toFuncFunctorArgs;
+
+
+            FuncExpr::Ptr from_func;
+            FuncExpr::Ptr to_func;
```

Comment:
this might not be nullptr by default. We should still initialize this to either nullptr